scripts/update-ruby.sh

```diff
@@ -23,8 +23,16 @@ die() {
     exit 1
 }
 
+# This assume that this script is invoked from the root of
+# react-native. The only place where this is invoked is the
+# `set-rn-version.js` script which invoked the script as
+# scripts/update-ruby.sh.
+MANAGED_RUBY_VERSION=$(cat .ruby-version)
+
 if [ $# -eq 1 ]; then
     VERSION=$1
+elif [ -n "$MANAGED_RUBY_VERSION" ]; then
+    VERSION="$MANAGED_RUBY_VERSION"
```

Comment:
@cipolleschi we don't trigger this alert for the user https://github.com/facebook/react-native/pull/35152/files#diff-b09bf54ec14d8fec6c73ad80cfd099ce37c34668b45830d182f312e59c6d851cL33

```bash
if [ -z "$VERSION" ]; then
    die "Please provide an installed/usable Ruby version"
fi
```

the script would stop execution and alert the user to install the correct ruby version. 
$VERSION has always a value.

Reply:
I'm wrong sorry. The check is at line https://github.com/facebook/react-native/blob/0a3ca80af401654896cbc73e235711eef0f9b3c5/scripts/update-ruby.sh#L47-L49

and it works as shown in https://app.circleci.com/pipelines/github/facebook/react-native/16984/workflows/ce8b02e1-b697-46cd-bdbf-119bbaaea0f9/jobs/334149

<img width="781" alt="Screen Shot 2022-10-31 at 2 40 51 PM" src="https://user-images.githubusercontent.com/24992535/199021554-75b4143a-aa5f-485c-98b3-9dae7acd620b.png">

The issue is that we need to upgrade the ubuntu version in circleci.

<img width="1195" alt="Screen Shot 2022-10-31 at 2 41 21 PM" src="https://user-images.githubusercontent.com/24992535/199021666-bcb769ba-bad8-49c0-b96a-812694e36044.png">

Reply:
Thanks! I'm trying to bump the ubuntu image first!